tests/test_api_models.py

```diff
@@ -81,6 +89,13 @@ def test_resourcelist(ocx, api_data):
         tags1.append(sample)
     assert 'object of type' in str(e.value)
 
+    if sys.version_info.major < 3:
+        with pytest.raises(AttributeError):
```

Comment:
Just out of curiosity, why does this raise? Probably worth a comment for future us.

Reply:
Apparently lists in Python 2 do not have a clear() method